packages/cells/src/widget.ts

```diff
@@ -312,6 +312,17 @@ class Cell extends Widget {
   }
 
   /**
+   * Clone the cell, using the same model.
+   */
+  clone() {
```

Comment:
Missing a return type.

Reply:
I couldn't figure out to make the type `this`, so I added explicit types, which meant I needed to add an explicit clone method to RawCell to get the typing right.